angular/polyfills-es5.js

```diff
@@ -0,0 +1 @@
6QF:function(t,e,n){n("I+eb")({target:"Number",stat:!0},{isInteger:n("Xol8")})},iSVu:function(t,e,n){var r=n("xs3f"),o=Function.toString;"function"!=typeof r.inspectSource&&(r.inspectSource=function(t){return o.call(t)}),t.exports=r.inspectSource},inlA:function(t,e,n){"use strict";var r,o=n("I+eb"),i=n("Bs8V").f,a=n("UMSQ"),c=n("WjRb"),u=n("HYAF"),s=n("qxPZ"),f=n("xDBR"),l="".endsWith,p=Math.min,h=s("endsWith");o({target:"String",proto:!0,forced:!(!f&&!h&&(r=i(String.prototype,"endsWith"),r&&!r.writable)||h)},{endsWith:function(t){var e=String(u(this));c(t);var n=arguments.length>1?arguments[1]:void 0,r=a(e.length),o=void 0===n?r:p(a(n),r),i=String(t);return l?l.call(e,i,o):e.slice(o-i.length,o)===i}})},iqWW:function(t,e,n){"use strict";var r=n("ZUd8").charAt;t.exports=function(t,e,n){return e+(n?r(t,e).length:1)}},jrUv:function(t,e){var n=Math.expm1,r=Math.exp;t.exports=!n||n(10)>22025.465794806718||n(10)<22025.465794806718||-2e-17!=n(-2e-17)?function(t){return 0==(t=+t)?t:t>-1e-6&&t<1e-6?t+t*t/2:r(t)-1}:n},jt2F:function(t,e,n){n("dG/n")("matchAll")},kNcU:function(t,e,n){var r=n("I+eb"),o=Math.log,i=Math.LN2;r({target:"Math",stat:!0},{log2:function(t){return o(t)/i}})},kOOl:function(t,e){var n=0,r=Math.random();t.exports=function(t){return"Symbol("+String(void 0===t?"":t)+")_"+(++n+r).toString(36)}},kRJp:function(t,e,n){var r=n("g6v/"),o=n("m/L8"),i=n("XGwC");t.exports=r?function(t,e,n){return o.f(t,e,i(1,n))}:function(t,e,n){return t[e]=n,t}},kSko:function(t,e,n){n("I+eb")({target:"Number",stat:!0},{isNaN:function(t){return t!=t}})},kmMV:function(t,e,n){"use strict";var r,o,i=n("rW0t"),a=n("n3/R"),c=RegExp.prototype.exec,u=String.prototype.replace,s=c,f=(o=/b*/g,c.call(r=/a/,"a"),c.call(o,"a"),0!==r.lastIndex||0!==o.lastIndex),l=a.UNSUPPORTED_Y||a.BROKEN_CARET,p=void 0!==/()??/.exec("")[1];(f||p||l)&&(s=function(t){var e,n,r,o,a=this,s=l&&a.sticky,h=i.call(a),v=a.source,d=0,g=t;return s&&(-1===(h=h.replace("y","")).indexOf("g")&&(h+="g"),g=String(t).slice(a.lastIndex),a.lastIndex>0&&(!a.multiline||a.multiline&&"\n"!==t[a.lastIndex-1])&&(v="(?: "+v+")",g=" "+g,d++),n=new RegExp("^(?:"+v+")",h)),p&&(n=new RegExp("^"+v+"$(?!\\s)",h)),f&&(e=a.lastIndex),r=c.call(s?n:a,g),s?r?(r.input=r.input.slice(d),r[0]=r[0].slice(d),r.index=a.lastIndex,a.lastIndex+=r[0].length):a.lastIndex=0:f&&r&&(a.lastIndex=a.global?r.index+r[0].length:e),p&&r&&r.length>1&&u.call(r[0],n,(function(){for(o=1;o<arguments.length-2;o++)void 0===arguments[o]&&(r[o]=void 0)})),r}),t.exports=s},l2dK:function(t,e,n){"use strict";var r=n("I+eb"),o=n("hXpO");r({target:"String",proto:!0,forced:n("rwPt")("fontcolor")},{fontcolor:function(t){return o(this,"font","color",t)}})},lEou:function(t,e,n){n("dG/n")("toStringTag")},lMq5:function(t,e,n){var r=n("0Dky"),o=/#|\.prototype\./,i=function(t,e){var n=c[a(t)];return n==s||n!=u&&("function"==typeof e?r(e):!!e)},a=i.normalize=function(t){return String(t).replace(o,".").toLowerCase()},c=i.data={},u=i.NATIVE="N",s=i.POLYFILL="P";t.exports=i},ls
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Object prototype is read only, properties should not be added.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5245121)